pod/import_video/forms.py

```diff
@@ -77,8 +77,8 @@ def __init__(self, *args, **kwargs):
         self.filter_fields_admin()
         self.fields = add_placeholder_and_asterisk(self.fields)
 
-        # We don't change the user who uploaded the record
-        hidden_fields = ("uploaded_to_pod_by",)
+        # We don't change the user who uploaded the record or the state
```

Comment:
Je mettrais
`# We don't change the user who uploaded the record neither the state`
pour que la phrase ne donne pas l'impression que "the user uploaded the record or the state"

Reply:
OK fait